### PR TITLE
[docbot] Reviewed some component chapters

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

File: components/config/caching.rst [hunk 1]
```diff
@@ -1,26 +1,27 @@
 .. index::
    single: Config; Caching based on resources
 
-Caching Based on Resources
+Caching based on Resources
```
Comment (javiereguiluz) [inline components/config/caching.rst]: I don't understand the twisted logic behind title casing, but are we sure about this change?

File: components/config/resources.rst [hunk 2]
```diff
@@ -14,8 +14,9 @@ Loading Resources
 Locating Resources
 ------------------
 
-Loading the configuration normally starts with a search for resources – in
-most cases: files. This can be done with the :class:`Symfony\\Component\\Config\\FileLocator`::
+Loading the configuration normally starts with a search for resources –
```
Comment (javiereguiluz) [inline components/config/resources.rst]: This phrase (the original one) looks strange:

```
with a search for resources – in most cases: files.
```

What about:

```
with a search for resources, mostly files.
```

File: components/dependency_injection/compilation.rst [hunk 3]
```diff
@@ -96,8 +99,8 @@ The Extension must specify a ``getAlias`` method to implement the interface::
         }
     }
 
-For YAML configuration files specifying the alias for the Extension as a key
-will mean that those values are passed to the Extension's ``load`` method:
+For YAML configuration files specifying the alias for the Extension as a
```
Comment (javiereguiluz) [inline components/dependency_injection/compilation.rst]: `Extension` -> `extension` ?

**Comment by javiereguiluz** [comment]
:+1: thanks for taking care of these boring but useful changes.

File: components/config/caching.rst (hunk 1 above)
Comment (xabbuh) [inline components/config/caching.rst]: Yeah looks odd to me too (isn't "based" derived from "to base (sth. on sth.)"?)

**Comment by stof** [comment]
@WouterJ is docbot ready to be run as a CI service or not yet ?

File: components/class_loader/class_loader.rst [hunk 4]
```diff
@@ -39,12 +39,12 @@ is straightforward::
...
-    The autoloader is automatically registered in a Symfony application (see
-    ``app/autoload.php``).
+    The autoloader is automatically registered in a Symfony application
+    (see ``app/autoload.php``).
 
-Use the :method:`Symfony\\Component\\ClassLoader\\ClassLoader::addPrefix` or
-:method:`Symfony\\Component\\ClassLoader\\ClassLoader::addPrefixes` methods to
-register your classes::
+Use the :method:`Symfony\\Component\\ClassLoader\\ClassLoader::addPrefix`
+or :method:`Symfony\\Component\\ClassLoader\\ClassLoader::addPrefixes` methods
```
Comment (OskarStark) [inline components/class_loader/class_loader.rst]: i think ```methods``` should be ```method``` @WouterJ

File: components/dependency_injection/compilation.rst [hunk 5]
```diff
@@ -8,8 +8,8 @@ The service container can be compiled for various reasons. These reasons
 include checking for any potential issues such as circular references and
 making the container more efficient by resolving parameters and removing
 unused services. Also, certain features - like using
-:doc:`parent services </components/dependency_injection/parentservices>` -
-require the container to be compiled.
+:doc:`parent services </components/dependency_injection/parentservices>`
+- require the container to be compiled.
```
Comment (weaverryan) [inline components/dependency_injection/compilation.rst]: won't this make this line a bullet?

File: components/config/caching.rst (hunk 1 above)
Comment (weaverryan) [inline components/config/caching.rst]: Looked odd to me at first, looks ok to me now - and I verified that lowercase makes sense from one quick source. Mostly, I don't care either way :)

File: components/class_loader/class_loader.rst (hunk 4 above)
Comment (weaverryan) [inline components/class_loader/class_loader.rst]: +1 (even though it's unrelated to the PR)

File: components/config/resources.rst (hunk 2 above)
Comment (weaverryan) [inline components/config/resources.rst]: :+1:

**Comment by weaverryan** [comment]
:+1: from me after the few comments

File: components/dependency_injection/compilation.rst (hunk 5 above)
Comment (WouterJ) [inline components/dependency_injection/compilation.rst]: Nope :) http://pr-5550-6qmocelev2lwe.eu.platform.sh/components/dependency_injection/compilation.html (I love this new platform.sh thing!)

Comment (weaverryan) [inline components/dependency_injection/compilation.rst]: nice!

File: components/config/caching.rst (hunk 1 above)
Comment (xabbuh) [inline components/config/caching.rst]: I really get the feeling that we should think again about our capitalisation rule. :)

**Pull request merged**